### DISCUSSION TITLE
Android example: Error

### DISCUSSION THREAD

**Discussion by rjpikus** [ggml-org/whisper.cpp#730] [Q&A] [answered]
Do I need to update or add refs? I'm getting an int when I should be getting a int8x8_t vector in ggml.h. int8x8_t vx_0 = vzip1_s8(vb_0, vb_1) is returning an int for some reason.
@zeitsmatt were you able to get to the bottom of this? I am running active build variant release and abi arm64-v8a.

**Comment by zeitsmatt** [comment, marked as answer]
The errors are coming from the wrong variant of the arm toolchain building along with the correct target.  It runs on my phone in debug mode, but its not really creating an APK.  I think when I closed out android studio and brought it back up, the target dialog was available and responded appropriately to only target the toolchain that matches the one that was listed above and in the comments in the git-hub repo.  I'll have to check my machine when I get back to it.  Even thought I managed to stop it from building the incorrect toolchain by cargo cult style troubleshooting, I still can't get it to build an APK, and that's probably because I haven't messed with android studio in a while--my team was using another tool to do our mobile apps--and have to re-familiarize myself with the IDE.  So, yes I can run it on my phone, but no, I'm not able to build the APK in release mode.

**Reply by rjpikus** [reply]
And getting an android app into the play store is another story. I am using cargo cult as well as I'm new to android. Might as well build it from scratch and add the pieces I need, thank you for the detailed response.
